contracts/PetPark.sol

```diff
@@ -1,7 +1,134 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+
+    enum AnimalType {
+        None,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+    enum Gender {
+        None,
+        Male,
+        Female
+    }
+
+    mapping(AnimalType => uint) public animalCounts;
+    mapping(address => AnimalType) public isBorrowing;
+    mapping(address => uint) public userAge;
+    mapping(address => Gender) public userGender;
```

Comment:
Best to create a struct with the animalType, age and gender fields and use that as the value in the mapping. So instead of 3 mappings you will have just one.